ddsrouter_cmake/cmake/project/configure_project.cmake

```diff
@@ -63,6 +63,13 @@ macro(configure_project)
         read_version(${MODULE_VERSION_FILE_PATH})
     endif()
 
+    # Read git commit hash if not already set
```

Comment:
Could it be set by CMake option? If so it may be added to README in ddsrouter_cmake along with the rest of variables

Reply:
where exactly? to be passed how?  as a `-DMODULE_COMMIT_HASH="my-hardcoded-hash"`?